client/src/components/helpers/AvatarRenderer.js

```diff
@@ -15,13 +15,31 @@ const propTypes = {
 
 function AvatarRenderer({ picture, userName, isDonating, isTopContributor }) {
   const { t } = useTranslation();
+  const [isPictureValid, setIsPictureValid] = useState(true);
   let borderColor = borderColorPicker(isDonating, isTopContributor);
-  let isPlaceHolderImage =
-    /example.com|identicon.org/.test(picture) || picture === defaultUserImage;
+  const loadEvent = () => setIsPictureValid(true);
+  const errorEvent = () => setIsPictureValid(false);
+  let validationImage = document.createElement('img');
+  validationImage.src = picture;
+
+  useEffect(() => {
+    validationImage.onload = loadEvent();
+  }, [picture, validationImage.onload]);
+  useEffect(() => {
+    validationImage.onerror = errorEvent();
+  }, [validationImage.onerror]);
+
+  const isPlaceHolderImage = () => {
+    return (
+      !isPictureValid ||
+      /example.com|identicon.org/.test(picture) ||
+      picture === defaultUserImage
+    );
+  };
```

Comment:
```suggestion
  const isPlaceHolderImage = 
    !isPictureValid ||
    /example.com|identicon.org/.test(picture) ||
    picture === defaultUserImage;
```
Since this only depends on props and state, it will change each render and doesn't need to be a function.

Reply:
Corrected